core/src/test/scala/com/microsoft/azure/synapse/ml/recommendation/RankingAdapterSpec.scala

```diff
@@ -24,4 +39,18 @@ class RankingAdapterModelSpec extends RankingTestBase with TransformerFuzzing[Ra
   }
 
   override def reader: MLReadable[_] = RankingAdapterModel
+
+  override def assertDFEq(df1: DataFrame, df2: DataFrame)(implicit eq: Equality[DataFrame]): Unit = {
```

Comment:
unless i'm missing something this is the same as above. can we reduce code duplication?